### PR TITLE
Fix and add small comments

### PR DESCRIPTION
**Pull request by nventuro**

### CODE REVIEW

File: pkg/pool-weighted/contracts/smart/ManagedPool.sol [hunk 1]
```diff
@@ -48,6 +48,8 @@ import "./WeightCompression.sol";
  * IL-protection, and more.
  */
 contract ManagedPool is BaseWeightedPool, ReentrancyGuard {
+    // LiquidityBootstrappingPool change their weights over time: these periods are expected to be long enough (e.g.
+    // days) that any timestamp manipulation would achieve very little.
```
Comment (EndymionJkb) [inline pkg/pool-weighted/contracts/smart/ManagedPool.sol:52]:
```suggestion
    // LiquidityBootstrappingPools change their weights over time: these periods are expected to be long enough
    // (e.g., days) that any timestamp manipulation would achieve very little.
```

File: pkg/pool-weighted/contracts/smart/LiquidityBootstrappingPool.sol [hunk 2]
```diff
@@ -23,10 +23,11 @@ import "../BaseWeightedPool.sol";
 import "./WeightCompression.sol";
 
 /**
- * @dev Weighted Pool with mutable weights, designed to support V2 Liquidity Bootstrapping
+ * @dev Weighted Pool with mutable weights, designed to support V2 Liquidity Bootstrapping.
  */
 contract LiquidityBootstrappingPool is BaseWeightedPool, ReentrancyGuard {
-    // The Pause Window and Buffer Period are timestamp-based: they should not be relied upon for sub-minute accuracy.
+    // LiquidityBootstrappingPool change their weights over time: these periods are expected to be long enough (e.g.
+    // days) that any timestamp manipulation would achieve very little.
```
Comment (EndymionJkb) [inline pkg/pool-weighted/contracts/smart/LiquidityBootstrappingPool.sol:30]:
```suggestion
    // LiquidityBootstrappingPools change their weights over time: these periods are expected to be long enough
    // (e.g., days) that any timestamp manipulation would achieve very little.
```

File: pkg/pool-weighted/contracts/smart/LiquidityBootstrappingPool.sol [hunk 3]
```diff
@@ -23,10 +23,11 @@ import "../BaseWeightedPool.sol";
 import "./WeightCompression.sol";
 
 /**
- * @dev Weighted Pool with mutable weights, designed to support V2 Liquidity Bootstrapping
+ * @dev Weighted Pool with mutable weights, designed to support V2 Liquidity Bootstrapping.
```
Comment (EndymionJkb) [inline pkg/pool-weighted/contracts/smart/LiquidityBootstrappingPool.sol:26]: Do we still need to say "V2" at this point?

**Review by EndymionJkb — COMMENTED** [review]
Do we still need to say "V2" at this point?

File: pkg/pool-weighted/contracts/smart/ManagedPool.sol (hunk 1 above)
Comment (nventuro) [reply pkg/pool-weighted/contracts/smart/ManagedPool.sol:52]: Huh, interesting. [According to this article](https://jakubmarian.com/comma-after-i-e-and-e-g/), this is a British vs American thing. We're doing brit style throughout the entire process, so I'd keep things consistent here and go with the Queen's grammar.

**Pull request merged**